database/src/migrations/20230925103600_create_survey_observations.ts

```diff
@@ -0,0 +1,199 @@
+
+
+
+  ----------------------------------------------------------------------------------------
+  -- Create new views
+  ----------------------------------------------------------------------------------------
+
+  set search_path=biohub_dapi_v1;
+
+  create or replace view survey_observation as select * from biohub.survey_observation;
+
+
+
+  ----------------------------------------------------------------------------------------
+  -- Update api_delete_survey procedure
+  ----------------------------------------------------------------------------------------
+
+  set search_path=biohub;
+
+  CREATE OR REPLACE PROCEDURE api_delete_survey(p_survey_id integer)
+  LANGUAGE plpgsql
+  SECURITY DEFINER
+  AS $procedure$
+   -- *******************************************************************
+     -- Procedure: api_delete_survey
+     -- Purpose: deletes a survey and dependencies
+     --
+     -- MODIFICATION HISTORY
+     -- Person           Date        Comments
+     -- ---------------- ----------- --------------------------------------
+     -- shreyas.devalapurkar@quartech.com
+     --                  2021-06-18  initial release
+     -- charlie.garrettjones@quartech.com
+     --                  2021-06-21  added occurrence submission delete
+     -- charlie.garrettjones@quartech.com
+     --                  2021-09-21  added survey summary submission delete
+     -- kjartan.einarsson@quartech.com
+     --                  2022-08-28  added survey_vantage, survey_spatial_component, survey delete
+     -- charlie.garrettjones@quartech.com
+     --                  2022-09-07  changes to permit model
+     -- charlie.garrettjones@quartech.com
+     --                  2022-10-05  1.3.0 model changes
+     -- charlie.garrettjones@quartech.com
+     --                  2022-10-05  1.5.0 model changes, drop concept of occurrence deletion for published data
+     -- charlie.garrettjones@quartech.com
+     --                  2023-03-14  1.7.0 model changes
+     -- alfred.rosenthal@quartech.com
+     --                  2023-03-15  added missing publish tables to survey delete
+     -- curtis.upshall@quartech.com
+     --                  2023-04-28  change order of survey delete procedure
+     -- alfred.rosenthal@quartech.com
+     --                  2023-07-26  delete regions
+     -- curtis.upshall@quartech.com
+     --                  2023-08-24  delete partnerships
+     -- curtis.upshall@quartech.com
+     --                  2023-08-24  delete survey blocks and stratums and participation
+     -- curtis.upshall@quartech.com
+     --                  2023-09-25  delete survey observations
+     -- *******************************************************************
+     declare
+
+     begin
+      with occurrence_submissions as (select occurrence_submission_id from occurrence_submission where survey_id = p_survey_id), submission_spatial_components as (select submission_spatial_component_id from submission_spatial_component
```

Comment:
Maybe this function was just always like this but I feel like all lowercase SQL keywords hurts readability.

Reply:
It has always been like this (I think). While I generally agree, there is no guarentee that any future migrations will respect such a change, since by definition, migrations are one-off scripts that generally don't follow readibility conventions.